### DISCUSSION TITLE
isChecked on MenuItemOption does not seem to work

### DISCUSSION THREAD

**Discussion by Poolshark** [chakra-ui/chakra-ui#4220] [Q&A]
I have played around with `MenuOptionGroups` and tried to set the `isChecked` property on the `MenuItemOption`. Unfortunately this does not do anything.

My minimalized example is:
```
<MenuList>
  <MenuOptionGroup type="checkbox">
    <MenuOptionItem isChecked={ true }  value="one">One</MenuOptionItem>
  </MenuOptionGroup>
</MenuList>
```

Is this just me or is there something wrong?

### UPDATE ---
Since this has been moved from being an issue to the discussions, I have decided to create a **CodeSandbox** example. I am pretty sure this is a bug but maybe I just do not understand how to use `MenuOptionItem`'s properly.

Here's the Sandbox Example:
[![Edit async-platform-cuj3d](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/async-platform-cuj3d?fontsize=14&hidenavigation=1&theme=dark)

**Comment by AlonGvili** [comment]
it's using context under the hood i think, so if you pass to MenuOptionGroup defaultValue={["email"]} and remove the isChecked from the item you see it working correctly 
https://codesandbox.io/s/prod-butterfly-vctms?file=/src/App.js

**Reply by Poolshark** [reply]
Thanks for your answer!

Yes, I've managed to get the initial checkbox state of each `MenuOptionItem` by passing the value array to the `MenuOptionGroup`. Nonetheless, I do not understand what the `isChecked` prop is for then? I have also tried to use it as a **render prop** but this does not seem to work either (see updated [Code Sandbox](https://codesandbox.io/s/prod-butterfly-vctms?file=/src/App.js)).

Is this really the intended behaviour? If yes then I clearly did not understand properly.

**Reply by toptal-dave** [reply]
Did you ever figure out what `isChecked` is for?  It seems a bit useless to me...

**Reply by mustafaaden98** [reply]
How to clear the checked value by click of some button? I tried but as isChecked is not functional, so I am not able to control the checked state
